src/design/illustrations.stories.mdx

```diff
@@ -31,6 +33,21 @@ Images may represent more than one topic. For example, the `paintbrush` illustra
   {() => imageDeck({ images: featureImageData })}
 </Story>
 
+## Responsive fallback image
+
+An SVG that's used as a fallback image when an image request fails for whatever reason.
+
+What's unique about this image is its responsiveness. Depending on the size of the broken image, portions of the image repeat or fix to the side to fit the aspect ratio of the image rather than squashing or stretching.
+
+<Story name="Responsive fallback image" height="200px">
+  {() =>
+    responsiveImageDeck({
+      ratios: ['1 / 1', '16 / 9', '9 / 16'],
```

Comment:
Quick question, @AriannaChau! 🙂  

In Safari, the aspect ratios all look the same in the documentation page. Does that matter? Is this expected? 🤔 

<img width="1392" alt="Screen Shot 2021-09-09 at 2 33 25 PM" src="https://user-images.githubusercontent.com/459757/132765699-11928c10-0164-45c7-bb4e-b2918832f64e.png">

Reply:
AGH! No, thank you for bringing that to my attention!

Reply:
Oh, is it because `aspect-ratio` isn't supported in Safari?

https://caniuse.com/mdn-css_properties_aspect-ratio

Reply:
It looks like Safari doesn't support `aspect-ratio` (though it does in the Technology Preview so it should be rolling out soon)

I think that's fine for our pattern documentation site (though I'd be hesitant to use it on our actual site). I'm curious what others think though

Reply:
Cool, I'm fine leaving it as-is for the documentation site. 😉